lambdas/src/contact-retrieve-transcript/index.ts

```diff
@@ -28,31 +29,24 @@ const hrm_env = process.env.hrm_env;
 const ssmCacheConfigs = [
   {
     path: `/${hrm_env}/twilio`,
-    regex: /AUTH_TOKEN/,
+    regex: /auth_token/,
   },
   {
     path: `/${hrm_env}/s3`,
-    regex: /DOCS_BUCKET_NAME/,
+    regex: /docs_bucket_name/,
   },
 ];
 
 const processRecord = async (sqsRecord: SQSRecord) => {
-  console.dir(sqsRecord);
-  /**
-   * TODO: This is still based around sending messages with SNS instead of directly to SES. I think we should
-   * consider moving to sending SES directly from HRM, as that will simplify the message structure. And simplify
-   * processing of the DLQ if we are planning to use the completed queue as the DLQ in the first iteration. If we
-   * were sending a generic "contactSaved" event to SNS and letting sns handle figuring out if it should
-   * process a transcript or a recording based on the payload, the SNS message structure would seem more
-   * worth the complexity. But if we know at job creation time that the job what type of job is being sent
-   * and what jobs should run, we can just send the payload directly to SES and avoid the added SNS message
-   * complexity.
-   */
-  const snsMessage: SNSMessage = JSON.parse(sqsRecord.body);
-  const message = JSON.parse(snsMessage.Message);
-
-  const authToken = ssmCache.values[`/${hrm_env}/twilio/${message.accountSid}/AUTH_TOKEN`];
-  const docsBucketName = ssmCache.values[`/${hrm_env}/s3/${message.accountSid}/DOCS_BUCKET_NAME`];
+  const message = JSON.parse(sqsRecord.body);
```

Comment:
I also went ahead and migrated to assuming direct SQS messages would be sent from hrm-services based on our conversations in the last PR. I can revert this part if we are planning to hold off on that change.